modules/EnsEMBL/Web/Configuration/Info.pm

```diff
@@ -53,8 +53,9 @@ sub populate_tree {
   my %error_messages = EnsEMBL::Web::Constants::ERROR_MESSAGES;
 
   ## Redirect strains to the strain page for the parent species as long as we're not on the parent species
-  if ($self->hub->action ne 'Strains' && $species_defs->STRAIN_COLLECTION && $species_defs->SPECIES_STRAIN !~ /reference/) {
-    my $url = $self->hub->url({'species' => $species_defs->STRAIN_COLLECTION, 'action' => 'Strains'});
+  if ( $self->hub->action ne 'Strain' && $species_defs->STRAIN_GROUP && $species_defs->SPECIES_STRAIN !~ /reference/
```

Comment:
The first check (ne 'Strain') instead of ne 'Strains' is that intentional?

Reply:
Oh, well spotted! Definitely should be 'Strains'.

Reply:
That's done. Thanks.